### PR TITLE
Logo routing/robert allen

### PR DESCRIPTION
**Pull request by robertjallen**
### Description

**Invited Collaborator(s) to Review
- Added person to review relating to feature/bug
- If not available, ask another reviewer on team

**What work was done?**
- When a user clicks on the logo it links back to that user's dashboard

**Why was this work done?**
- When a user clicks on the logo it links back to that user's dashboard

**What feature / user story is it for?**
When a user clicks on the logo it links back to that user's dashboard
### Type of change
Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)

Testing
X Has this feature been tested before conflict?

Change Status
X Completed, ready to review and merge

### Checklist

XMy changes generate no new warnings
X There are no merge conflicts

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -55,10 +55,11 @@ function App() {
         <Route exact path="/login" component={LoginPage} />
         <Route path="/implicit/callback" component={LoginCallback} />
         {/* any of the routes you need secured should be registered as SecureRoutes */}
+
         <SecureRoute
           path="/"
           exact
-          component={() => <MyProfile LoadingOutlined={LoadingOutlined} />}
+          component={() => <DashboardPage LoadingOutlined={LoadingOutlined} />}
```
Comment (Monicascz) [inline src/index.js:62]: Nice! Good thinking to get the user to land on the Dashboard after login as opposed to their Profile page.

**Comment by Monicascz** [comment]
It looks like one of the checks is failing: build/Test and publish test coverage. Please take a look at the details and see if we can get that check to pass before merging

File: src/index.js (hunk 1 above)
Comment (JasonCorchado) [reply src/index.js:62]: Like Monica said Good idea changing the routing.

**Review by JasonCorchado — APPROVED** [review]

**Review by Monicascz — APPROVED** [review]
Great job with this! Changing the logo to route to the user's 'homepage' is what most users expect and was a very UX friendly change. Nice work!

**Pull request merged**